### PR TITLE
SW-703_product_listing_filter_container

### PR DESCRIPTION
**Pull request by dblaichinger**
…iner before product listing

### CODE REVIEW

File: src/Resources/views/storefront/page/search/layout/leftFilterLayout.html.twig [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+{% endif %}
+
+<div class="row">
+    <div class="cms-element-sidebar-filter {{ filterColumns }}">
+        <div id="filter-panel-wrapper"
+             class="filter-panel-wrapper"
+             data-offcanvas-filter-content="true">
+            {% sw_include '@Storefront/storefront/component/listing/filter-panel.html.twig' with {
+                listing: page.listing,
+                sidebar: isInSidebar
```
Comment (TobiasGraml11) [inline src/Resources/views/storefront/page/search/layout/leftFilterLayout.html.twig]: This is always true inside of leftFilterLayout, right? So I guess we can remove this parameter now. (Including the definition in search-pagelet)

File: src/Resources/views/storefront/page/search/layout/topFilterLayout.html.twig [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<div class="cms-element-sidebar-filter">
+    <div id="filter-panel-wrapper"
+         class="filter-panel-wrapper"
+         data-offcanvas-filter-content="true">
+        {% sw_include '@Storefront/storefront/component/listing/filter-panel.html.twig' with {
+            listing: page.listing,
+            sidebar: isInSidebar
```
Comment (TobiasGraml11) [inline src/Resources/views/storefront/page/search/layout/topFilterLayout.html.twig]: `sidebar` will always need `false` for the top layout

File: src/Resources/views/storefront/page/search/search-pagelet.html.twig [hunk 3]
```diff
@@ -21,39 +19,9 @@
         {% if page.header.extensions.flConfig.filterPosition is constant('\\FINDOLOGIC\\FinSearch\\Findologic\\FilterPosition::LEFT') %}
             {% set sidebarClasses = 'has--sidebar' %}
```
Comment (TobiasGraml11) [inline src/Resources/views/storefront/page/search/search-pagelet.html.twig]: Where is this variable even used?

**Review by TobiasGraml11 — CHANGES_REQUESTED** [review]

File: src/Resources/views/storefront/page/search/search-pagelet.html.twig (hunk 3 above)
Comment (dblaichinger) [reply src/Resources/views/storefront/page/search/search-pagelet.html.twig]: Used in Shopware templates, but it's also set there so I don't see any reason to do it here.

File: src/Resources/views/storefront/page/search/search-pagelet.html.twig [hunk 4]
```diff
@@ -2,58 +2,23 @@
 
 {% set listingColumns = 'col-sm-6 col-lg-4 col-xl-3' %}
```
Comment (TobiasGraml11) [inline src/Resources/views/storefront/page/search/search-pagelet.html.twig]: When we add all those other variables inside the enabled check, should we also move this there?

Comment (TobiasGraml11) [reply src/Resources/views/storefront/page/search/search-pagelet.html.twig]: Or why is this even needed? If it is something from Shopware, I guess this should also work without us setting it again

Comment (dblaichinger) [reply src/Resources/views/storefront/page/search/search-pagelet.html.twig]: Seems like it's not necessary, classes are still set when removed
Screen Shot 2022-04-07 at 10 08 47
:

**Review by TobiasGraml11 — APPROVED** [review]

**Pull request merged**